tools/autograd/load_derivatives.py

```diff
@@ -169,10 +169,11 @@ def find_required_inputs(formula: str, postfix: str) -> Tuple[str, ...]:
                                    "forward definition of gradient as element_wise but it does not "
                                    "defines the gradient formula for its argument which is required.")
             # This transformation is based on the observation that for element-wise functions, the Jacobian
-            # matrix is diagonal and thus doing J * v or v * J gives the same result.
+            # matrix is diagonal and thus doing J * v or v * J.conj() or (v.conj() J).conj() gives the same result.
```

Comment:
I agree that `v * J.conj()` and `(v.conj() J).conj()` are the same, but are those really the same as `J * v` ?

Reply:
I think it meant to be `J.conj() * v`

Reply:
No the idea is really to say that we want to do `J * v` and every transposition is replaced by a hermitian transpose.
Since J is diagonal, it doesn't move but v and the result are hermitian transposed, hence the `(v^H J)^H`. Ignoring the tranpose again leaves just the conjugates.